README.md

```diff
@@ -36,7 +36,7 @@ Use `--help` with each command for more details.
 
 `parse` to parse a Stable MIR JSON file (`*.smir.json`) file to a K AST
 
-`kmir run` to load an SMIR JSON generated by the `stable-mir-json` tool.
+`kmir run` to run a binary target in your cargo project. The SMIR JSON will need to have been already built with `stable-mir-json`, or you will need the `RUSTC` environment variable set to your installation of `stable-mir-json`.
```

Comment:
Is it _required_ for `kmir run` to have a `cargo` project now?
(I don't think it is, and I would not want it that way TBH). Maybe reformulate this and give an example?

Reply:
Yes, this makes it required. I can write a dev script that still works with a `.smir.json` argument if you're still trying to work that way. We've been discussing making the CLI interface more like using cargo, this is meant to be a step in that direction.

Reply:
I think we could have a second option (say `--file`) which, when provided, would not use the `cargo` global at all.
Not everyone who writes a small Rust program wants to, nor has to, use cargo so  I think it would make sense to keep around a way to simply run a given `*.smir.json` file.

Reply:
```suggestion
`kmir run` to run a binary target in your cargo project (or a provided `*.smir.json` file).
For `cargo, the SMIR JSON will need to have been already built with `stable-mir-json`,
or you will need the `RUSTC` environment variable set to your installation of `stable-mir-json`.
```